### PR TITLE
Stubfor cambridge bfbs171

### PR DESCRIPTION
**Pull request by karljonaskarlsson**
Created stub

### CODE REVIEW

**Review by thea-m — APPROVED** [review]
From the technical side, everything seems fine!

**Comment by SophiaD-M** [comment]
Identifiers wrong - two different signatures, please change

File: LondonBibleSociety/BSLet171.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            <p>Encoded according to TEI P5 Guidelines.</p>
+         </projectDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc>
+         <langUsage><language ident="en">English</language></langUsage>
```
Comment (SophiaD-M) [inline LondonBibleSociety/BSLet171.xml:69]: maybe add also Geez

File: LondonBibleSociety/BSLet171.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                  licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+                  <msIdentifier>
+                     <repository ref="INS0306BSL"/>
+                     <idno>BFBS Mss 171</idno>
```
Comment (SophiaD-M) [inline LondonBibleSociety/BSLet171.xml:30]: add all known alternative signatures

**Review by SophiaD-M — CHANGES_REQUESTED** [review]

**Review by SophiaD-M — APPROVED** [review]

**Pull request merged**